### ISSUE TITLE
stop ingest on index failure (with read-only index)

### ISSUE THREAD

**Issue by jalogisch** [Graylog2/graylog2-server#5812]
## Current Behavior
Graylog does not give any indication on index failures in the UI that this had happened. 

In the log of Graylog you might find the following:
```
2019-03-26T20:00:19.082Z WARN [Messages] Failed to index message: index=<graylog_110> id= error=<{“type”:“cluster_block_exception”,“reason”:“blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];”}>
2019-03-26T20:00:19.082Z WARN [Messages] Failed to index message: index=<graylog_110> id= error=<{“type”:“cluster_block_exception”,“reason”:“blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];”}>
2019-03-26T20:00:19.082Z ERROR [Messages] Failed to index [500] messages. Please check the index error log in your web interface for the reason. Error: One or more of the items in the Bulk request failed, check BulkResult.getItems() for more information.cd
```

The real reason behind is that Elasticsearch (6) now makes the index read-only when the configured watermark is reached. 

## Context
While the behaviour of Elasticsearch is nice and might prefend people run into full filesystems, Graylog can't handle this situation and messages will be lost. 
The Cluster Status is `green` but all indices are read-only, that is something we need to be able to identify and stop ingestion and make it visible to the user in the Graylog UI.

## Your Environment

* Graylog Version: 3.0
* Elasticsearch Version: 6

**Comment by jalogisch** [comment]
We should not only warn on index read only, the ingest should stop as long as the write index is read-only.

**Comment by jameskirsop** [comment]
Further, it would be helpful if the `is_processing` API response returned false in this instance, so external monitoring systems can generate appropriate alerts.

I'm not clear if 'the ingest should stop' equates to a false `is_processing` response.

**Comment by dennisoelkers** [comment]
Fixed in #8245.
